tengo_test.go

```diff
@@ -327,6 +327,65 @@ func anotherTableForFlavor(flavor Flavor) Table {
 	return table
 }
 
+func aSpatialTable() Table {
+	return anotherTableForFlavor(FlavorMySQL80)
+}
+
+func spatialTableForFlavor(flavor Flavor) Table {
+	columns := []*Column{
+		{
+			Name:     "actor_id",
+			TypeInDB: "smallint(5) unsigned",
+		},
+		{
+			Name:     "latlon_p",
+			TypeInDB: "POINT",
+		},
+		{
+			Name:     "latlon_g",
+			TypeInDB: "GEOMETRY",
+			SRID:     "4326",
+		},
+	}
+	pointIndex := &Index{
+		Name: "p_idx",
+		Parts: []IndexPart{
+			{ColumnName: columns[1].Name},
+		},
+		Type: "SPATIAL",
+	}
+	geometryIndex := &Index{
+		Name: "g_idx",
+		Parts: []IndexPart{
+			{ColumnName: columns[2].Name},
+		},
+		Type: "SPATIAL",
+	}
+	stmt := `CREATE TABLE ` + "`" + `actor_spatial` + "`" + ` (
+  ` + "`" + `actor_id` + "`" + ` smallint(5) unsigned NOT NULL,
+  ` + "`" + `latlon_p` + "`" + ` point NOT NULL,
+  ` + "`" + `latlon_g` + "`" + ` geometry NOT NULL srid 4326,
+  PRIMARY KEY (` + "`" + `actor_id` + "`" + `),
+  SPATIAL KEY (` + "`" + `latlon_p` + "`" + `),
+  SPATIAL KEY (` + "`" + `latlon_g` + "`" + `),
+) ENGINE=InnoDB DEFAULT CHARSET=latin1`
```

Comment:
Silly nit but if you simply quote everything here with `"` you can simplify the declaration a lot and remove many of the `+` operators. There's no reason why `CREATE TABLE ` needs to be quoted with backticks.

Reply:
I have merely complied with the coding convention used in similar tests. I do not know why this is the convention used, but I think it's best to stick to it if this is to be contributed upstream.

Reply:
FWIW the reason this was done this way is that `"` does not support line breaks. However, further down the test file there was another method used, which I adopted.